docroot/themes/humsci/su_humsci_theme/js/hover_menu.js

```diff
@@ -29,25 +29,67 @@
         }
       }
 
+      // Check main menu items to ensure dropdown submenus don't get orphaned or cut off.
+      function menuEdgeCheck() {
+        var viewportWidth = document.body.clientWidth;
+        jQuery('ul.decanter-nav-primary > li').each(function() {
```

Comment:
Should add `context` to the selector function

Reply:
@pookmish I am not sure what you mean.

Reply:
Please review https://www.drupal.org/docs/8/api/javascript-api/javascript-api-overview under the "Drupal.behaviors" heading

Reply:
OK, done.